src/lib/viewers/doc/__tests__/DocBaseViewer-test.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-unused-expressions */
 import React from 'react';
+import { bdlBoxBlue } from 'box-ui-elements/es/styles/variables';
```

Comment:
Do we need this specific color for the test or can we use a generic color?

Reply:
Nope, we can use a generic color.